Jenkins job failed with stack dump for getpan

Interesting to note that the code changes that triggered the build had not touched the **cpanfile** and getpan has worked well  prior to this. We always do a clean build (workspace scrubbed prior to checkout and getpan and cpanm cache directories homed under the workspace).


```bash
/home/jenkinsworkspace/bin/getpan -cpus 1 -cachedir=/home/jenkinsworkspace/workspace/env/sat/ltest/chs/developer/build/.gopancache -smart http://wswebdev3.orctel.internal:5888 -loglevel=INFO -nodepdump -nocpan -nobackpan -metacpan -installdir grunt_mods Text::Xslate Tie::IxHash Digest::SHA1; \
	/home/jenkinsworkspace/bin/getpan -cpus 1 -cachedir=/home/jenkinsworkspace/workspace/env/sat/ltest/chs/developer/build/.gopancache -smart http://wswebdev3.orctel.internal:5888 -loglevel=INFO -nodepdump -nocpan -nobackpan -metacpan
[2015-01-19 13:14:28.797324579 +0000 GMT] [INFO] Installing from command line args
[2015-01-19 13:14:28.797495814 +0000 GMT] [INFO] Query: http://wswebdev3.orctel.internal:5888/where/Text::Xslate/>0.00
[2015-01-19 13:14:28.800319299 +0000 GMT] [INFO] Module not found in SmartPAN: Text::Xslate
[2015-01-19 13:14:30.770791249 +0000 GMT] [INFO] => Data::MessagePack (0.380000)
[2015-01-19 13:14:30.770877228 +0000 GMT] [INFO] => Mouse (0.610000)
[2015-01-19 13:14:30.770910627 +0000 GMT] [INFO] => Scalar::Util (1.140000)
[2015-01-19 13:14:30.770953304 +0000 GMT] [INFO] => XSLoader (0.000000)
[2015-01-19 13:14:30.770998946 +0000 GMT] [INFO] => parent (0.221000)
[2015-01-19 13:14:30.771056776 +0000 GMT] [INFO] => perl (5.810000)
[2015-01-19 13:14:30.771138771 +0000 GMT] [INFO] Query: http://wswebdev3.orctel.internal:5888/where/Data::MessagePack/>0.00
[2015-01-19 13:14:30.772155498 +0000 GMT] [INFO] Module not found in SmartPAN: Data::MessagePack
[2015-01-19 13:14:31.626499604 +0000 GMT] [INFO] => XSLoader (0.000000)
[2015-01-19 13:14:31.626586021 +0000 GMT] [INFO] => perl (5.810000)
[2015-01-19 13:14:31.626653479 +0000 GMT] [INFO] Query: http://wswebdev3.orctel.internal:5888/where/Mouse/>0.00
[2015-01-19 13:14:31.627701757 +0000 GMT] [INFO] Module not found in SmartPAN: Mouse
[2015-01-19 13:14:33.6653812 +0000 GMT] [INFO] => Scalar::Util (0.000000)
[2015-01-19 13:14:33.665462001 +0000 GMT] [INFO] => XSLoader (0.000000)
[2015-01-19 13:14:33.665516678 +0000 GMT] [INFO] => perl (0.000000)
[2015-01-19 13:14:33.665589767 +0000 GMT] [INFO] Query: http://wswebdev3.orctel.internal:5888/where/Tie::IxHash/>0.00
[2015-01-19 13:14:33.666483942 +0000 GMT] [INFO] Module not found in SmartPAN: Tie::IxHash
[2015-01-19 13:14:34.215390878 +0000 GMT] [INFO] => perl (0.000000)
[2015-01-19 13:14:34.215493834 +0000 GMT] [INFO] Query: http://wswebdev3.orctel.internal:5888/where/Digest::SHA1/>0.00
[2015-01-19 13:14:34.216270719 +0000 GMT] [INFO] Module not found in SmartPAN: Digest::SHA1
[2015-01-19 13:14:34.848113556 +0000 GMT] [INFO] => Digest::base (0.000000)
[2015-01-19 13:14:34.84820333 +0000 GMT] [INFO] => perl (0.000000)
[2015-01-19 13:14:36.284465101 +0000 GMT] [INFO] Installed Tie-IxHash (1.23)
[2015-01-19 13:14:38.872829837 +0000 GMT] [INFO] Installed Digest-SHA1 (2.13)
[2015-01-19 13:14:44.041864064 +0000 GMT] [INFO] Installed Data-MessagePack (0.48)
[2015-01-19 13:15:08.722072131 +0000 GMT] [INFO] Installed Mouse (2.4.1)
[2015-01-19 13:15:17.358725204 +0000 GMT] [INFO] Installed Text-Xslate (3.3.3)
[2015-01-19 13:15:17.358839968 +0000 GMT] [INFO] Successfully installed 5 modules
[2015-01-19 13:15:17.365458055 +0000 GMT] [INFO] Installing from cpanfile: cpanfile
[2015-01-19 13:15:17.366098801 +0000 GMT] [INFO] Mojolicious (== 5.47)
[2015-01-19 13:15:17.366131366 +0000 GMT] [INFO] => Found dependency: Mojolicious (== 5.47)
[2015-01-19 13:15:17.366188813 +0000 GMT] [INFO] Mango (> 0.00)
[2015-01-19 13:15:17.366214204 +0000 GMT] [INFO] => Found dependency: Mango (> 0.00)
[2015-01-19 13:15:17.366258388 +0000 GMT] [INFO] Test::Mock::Mango (> 0.00)
[2015-01-19 13:15:17.366282802 +0000 GMT] [INFO] => Found dependency: Test::Mock::Mango (> 0.00)
[2015-01-19 13:15:17.366328266 +0000 GMT] [INFO] AnyEvent (== 5.29)
[2015-01-19 13:15:17.366352743 +0000 GMT] [INFO] => Found dependency: AnyEvent (== 5.29)
[2015-01-19 13:15:17.366398183 +0000 GMT] [INFO] JSON::XS (> 0.00)
[2015-01-19 13:15:17.366422729 +0000 GMT] [INFO] => Found dependency: JSON::XS (> 0.00)
[2015-01-19 13:15:17.366463736 +0000 GMT] [INFO] MojoX::JSON::XS (> 0.00)
[2015-01-19 13:15:17.366487631 +0000 GMT] [INFO] => Found dependency: MojoX::JSON::XS (> 0.00)
[2015-01-19 13:15:17.366563401 +0000 GMT] [INFO] MojoX::Plugin::PODRenderer (> 0.00)
[2015-01-19 13:15:17.366588913 +0000 GMT] [INFO] => Found dependency: MojoX::Plugin::PODRenderer (> 0.00)
[2015-01-19 13:15:17.366642822 +0000 GMT] [INFO] MojoX::Plugin::Statsd (> 0.00)
[2015-01-19 13:15:17.366667431 +0000 GMT] [INFO] => Found dependency: MojoX::Plugin::Statsd (> 0.00)
[2015-01-19 13:15:17.366707486 +0000 GMT] [INFO] Text::Xslate (> 0.00)
[2015-01-19 13:15:17.366731513 +0000 GMT] [INFO] => Found dependency: Text::Xslate (> 0.00)
[2015-01-19 13:15:17.366780617 +0000 GMT] [INFO] Crypt::CBC (> 0.00)
[2015-01-19 13:15:17.366804169 +0000 GMT] [INFO] => Found dependency: Crypt::CBC (> 0.00)
[2015-01-19 13:15:17.366846142 +0000 GMT] [INFO] MojoX::Renderer::Xslate (> 0.00)
[2015-01-19 13:15:17.366869755 +0000 GMT] [INFO] => Found dependency: MojoX::Renderer::Xslate (> 0.00)
[2015-01-19 13:15:17.366931542 +0000 GMT] [INFO] YAML::XS (> 0.00)
[2015-01-19 13:15:17.366955437 +0000 GMT] [INFO] => Found dependency: YAML::XS (> 0.00)
[2015-01-19 13:15:17.367022336 +0000 GMT] [INFO] Crypt::Rijndael (> 0.00)
[2015-01-19 13:15:17.367047392 +0000 GMT] [INFO] => Found dependency: Crypt::Rijndael (> 0.00)
[2015-01-19 13:15:17.367084649 +0000 GMT] [INFO] CH::Log (> 0.00)
[2015-01-19 13:15:17.367108405 +0000 GMT] [INFO] => Found dependency: CH::Log (> 0.00)
[2015-01-19 13:15:17.367604426 +0000 GMT] [INFO] CH::Exception (> 0.00)
[2015-01-19 13:15:17.367650131 +0000 GMT] [INFO] => Found dependency: CH::Exception (> 0.00)
[2015-01-19 13:15:17.367699423 +0000 GMT] [INFO] CH::Perl (> 0.00)
[2015-01-19 13:15:17.367727749 +0000 GMT] [INFO] => Found dependency: CH::Perl (> 0.00)
[2015-01-19 13:15:17.367765666 +0000 GMT] [INFO] CH::Test (> 0.00)
[2015-01-19 13:15:17.367789773 +0000 GMT] [INFO] => Found dependency: CH::Test (> 0.00)
[2015-01-19 13:15:17.367836741 +0000 GMT] [INFO] CH::MojoX::Plugin::Config (> 0.00)
[2015-01-19 13:15:17.367861218 +0000 GMT] [INFO] => Found dependency: CH::MojoX::Plugin::Config (> 0.00)
[2015-01-19 13:15:17.367909328 +0000 GMT] [INFO] MooseX::Model (> 0.00)
[2015-01-19 13:15:17.367932686 +0000 GMT] [INFO] => Found dependency: MooseX::Model (> 0.00)
[2015-01-19 13:15:17.367973237 +0000 GMT] [INFO] MangoX::EventWrapper (> 0.00)
[2015-01-19 13:15:17.368003003 +0000 GMT] [INFO] => Found dependency: MangoX::EventWrapper (> 0.00)
[2015-01-19 13:15:17.36804832 +0000 GMT] [INFO] Mojolicious::Plugin::MangoWrapper (> 0.00)
[2015-01-19 13:15:17.368071393 +0000 GMT] [INFO] => Found dependency: Mojolicious::Plugin::MangoWrapper (> 0.00)
[2015-01-19 13:15:17.36811377 +0000 GMT] [INFO] CH::MojoX::Plugin::CORS (> 0.00)
[2015-01-19 13:15:17.368137356 +0000 GMT] [INFO] => Found dependency: CH::MojoX::Plugin::CORS (> 0.00)
[2015-01-19 13:15:17.368177328 +0000 GMT] [INFO] MojoX::OAuth2 (> 0.00)
[2015-01-19 13:15:17.368200815 +0000 GMT] [INFO] => Found dependency: MojoX::OAuth2 (> 0.00)
[2015-01-19 13:15:17.36824576 +0000 GMT] [INFO] CH::MojoX::SignIn::Bridge::OAuth2 (> 0.00)
[2015-01-19 13:15:17.368268614 +0000 GMT] [INFO] => Found dependency: CH::MojoX::SignIn::Bridge::OAuth2 (> 0.00)
[2015-01-19 13:15:17.368411731 +0000 GMT] [INFO] JSON::WebEncryption (> 0.00)
[2015-01-19 13:15:17.368435346 +0000 GMT] [INFO] => Found dependency: JSON::WebEncryption (> 0.00)
[2015-01-19 13:15:17.36849621 +0000 GMT] [INFO] MojoX::Security::Session (> 0.00)
[2015-01-19 13:15:17.368519387 +0000 GMT] [INFO] => Found dependency: MojoX::Security::Session (> 0.00)
[2015-01-19 13:15:17.368558102 +0000 GMT] [INFO] Locale::Simple (> 0.00)
[2015-01-19 13:15:17.368581442 +0000 GMT] [INFO] => Found dependency: Locale::Simple (> 0.00)
[2015-01-19 13:15:17.368627122 +0000 GMT] [INFO] MojoX::Plugin::Hook::BeforeRendered (> 0.00)
[2015-01-19 13:15:17.368650282 +0000 GMT] [INFO] => Found dependency: MojoX::Plugin::Hook::BeforeRendered (> 0.00)
[2015-01-19 13:15:17.368695435 +0000 GMT] [INFO] Log::Log4perl::Appender::Fluent (> 0.00)
[2015-01-19 13:15:17.368718394 +0000 GMT] [INFO] => Found dependency: Log::Log4perl::Appender::Fluent (> 0.00)
[2015-01-19 13:15:17.368760096 +0000 GMT] [INFO] MojoX::Plugin::AnyCache (> 0.00)
[2015-01-19 13:15:17.368782716 +0000 GMT] [INFO] => Found dependency: MojoX::Plugin::AnyCache (> 0.00)
[2015-01-19 13:15:17.368827478 +0000 GMT] [INFO] CH::MojoX::Administration::Plugin (> 0.00)
[2015-01-19 13:15:17.368850842 +0000 GMT] [INFO] => Found dependency: CH::MojoX::Administration::Plugin (> 0.00)
[2015-01-19 13:15:17.368896701 +0000 GMT] [INFO] CH::MojoX::UserPermissions::Bridge (> 0.00)
[2015-01-19 13:15:17.368919378 +0000 GMT] [INFO] => Found dependency: CH::MojoX::UserPermissions::Bridge (> 0.00)
[2015-01-19 13:15:17.368962193 +0000 GMT] [INFO] CH::MojoX::Plugin::Xslate (> 0.00)
[2015-01-19 13:15:17.368989742 +0000 GMT] [INFO] => Found dependency: CH::MojoX::Plugin::Xslate (> 0.00)
[2015-01-19 13:15:17.369047137 +0000 GMT] [INFO] Protocol::Redis::XS (> 0.00)
[2015-01-19 13:15:17.369070822 +0000 GMT] [INFO] => Found dependency: Protocol::Redis::XS (> 0.00)
[2015-01-19 13:15:17.369097408 +0000 GMT] [INFO] Found 34 dependencies in cpanfile
[2015-01-19 13:15:17.369328262 +0000 GMT] [INFO] Query: http://wswebdev3.orctel.internal:5888/where/Mojolicious/==5.47
[2015-01-19 13:15:17.372082897 +0000 GMT] [INFO] Module not found in SmartPAN: Mojolicious
[2015-01-19 13:15:22.446934762 +0000 GMT] [INFO] => Pod::Simple (3.090000)
[2015-01-19 13:15:22.447025865 +0000 GMT] [INFO] => Time::Local (1.200000)
[2015-01-19 13:15:22.447061486 +0000 GMT] [INFO] => perl (5.010001)
[2015-01-19 13:15:22.447128746 +0000 GMT] [INFO] Query: http://wswebdev3.orctel.internal:5888/where/Mango/>0.00
[2015-01-19 13:15:22.448113256 +0000 GMT] [INFO] Module not found in SmartPAN: Mango
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x10 pc=0x457cac]

goroutine 22 [running]:
runtime.panic(0x7e7840, 0xa68173)
	/usr/local/Cellar/go/1.3.1/libexec/src/pkg/runtime/panic.c:279 +0xf5
github.com/ian-kent/gopan/getpan/getpan.(*Source).Find(0xc2080b8640, 0xc20801b3b0, 0x0, 0x0, 0x0)
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/sources.go:219 +0x220c
github.com/ian-kent/gopan/getpan/getpan.(*Dependency).Resolve(0xc20801b3b0, 0x0, 0x0)
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:331 +0x27d
github.com/ian-kent/gopan/getpan/getpan.func��003(0xc20801b3b0)
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:224 +0x3e3
created by github.com/ian-kent/gopan/getpan/getpan.(*DependencyList).Resolve
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:310 +0x470

goroutine 16 [semacquire]:
sync.runtime_Semacquire(0xc208000d40)
	/usr/local/Cellar/go/1.3.1/libexec/src/pkg/runtime/sema.goc:199 +0x30
sync.(*WaitGroup).Wait(0xc2080bab00)
	/usr/local/Cellar/go/1.3.1/libexec/src/pkg/sync/waitgroup.go:129 +0x14b
github.com/ian-kent/gopan/getpan/getpan.(*DependencyList).Resolve(0xc2080ba5e0, 0x0, 0x0)
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:313 +0x49f
main.main()
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/main.go:86 +0xe22

goroutine 19 [finalizer wait]:
runtime.park(0x414730, 0xa6ccb8, 0xa6aee9)
	/usr/local/Cellar/go/1.3.1/libexec/src/pkg/runtime/proc.c:1369 +0x89
runtime.parkunlock(0xa6ccb8, 0xa6aee9)
	/usr/local/Cellar/go/1.3.1/libexec/src/pkg/runtime/proc.c:1385 +0x3b
runfinq()
	/usr/local/Cellar/go/1.3.1/libexec/src/pkg/runtime/mgc0.c:2644 +0xcf
runtime.goexit()
	/usr/local/Cellar/go/1.3.1/libexec/src/pkg/runtime/proc.c:1445

goroutine 23 [chan send]:
github.com/ian-kent/gopan/getpan/getpan.func��003(0xc20801b400)
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:211 +0xaa
created by github.com/ian-kent/gopan/getpan/getpan.(*DependencyList).Resolve
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:310 +0x470

goroutine 24 [chan send]:
github.com/ian-kent/gopan/getpan/getpan.func��003(0xc20801b4a0)
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:211 +0xaa
created by github.com/ian-kent/gopan/getpan/getpan.(*DependencyList).Resolve
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:310 +0x470

goroutine 25 [chan send]:
github.com/ian-kent/gopan/getpan/getpan.func��003(0xc20801b540)
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:211 +0xaa
created by github.com/ian-kent/gopan/getpan/getpan.(*DependencyList).Resolve
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:310 +0x470

goroutine 26 [chan send]:
github.com/ian-kent/gopan/getpan/getpan.func��003(0xc20801b5e0)
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:211 +0xaa
created by github.com/ian-kent/gopan/getpan/getpan.(*DependencyList).Resolve
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:310 +0x470

goroutine 27 [chan send]:
github.com/ian-kent/gopan/getpan/getpan.func��003(0xc20801b680)
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:211 +0xaa
created by github.com/ian-kent/gopan/getpan/getpan.(*DependencyList).Resolve
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:310 +0x470

goroutine 28 [chan send]:
github.com/ian-kent/gopan/getpan/getpan.func��003(0xc20801b6d0)
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:211 +0xaa
created by github.com/ian-kent/gopan/getpan/getpan.(*DependencyList).Resolve
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:310 +0x470

goroutine 29 [chan send]:
github.com/ian-kent/gopan/getpan/getpan.func��003(0xc20801b770)
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:211 +0xaa
created by github.com/ian-kent/gopan/getpan/getpan.(*DependencyList).Resolve
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:310 +0x470

goroutine 30 [chan send]:
github.com/ian-kent/gopan/getpan/getpan.func��003(0xc20801b810)
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:211 +0xaa
created by github.com/ian-kent/gopan/getpan/getpan.(*DependencyList).Resolve
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:310 +0x470

goroutine 31 [chan send]:
github.com/ian-kent/gopan/getpan/getpan.func��003(0xc20801b8b0)
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:211 +0xaa
created by github.com/ian-kent/gopan/getpan/getpan.(*DependencyList).Resolve
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:310 +0x470

goroutine 32 [chan send]:
github.com/ian-kent/gopan/getpan/getpan.func��003(0xc20801b900)
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:211 +0xaa
created by github.com/ian-kent/gopan/getpan/getpan.(*DependencyList).Resolve
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:310 +0x470

goroutine 33 [chan send]:
github.com/ian-kent/gopan/getpan/getpan.func��003(0xc20801b9a0)
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:211 +0xaa
created by github.com/ian-kent/gopan/getpan/getpan.(*DependencyList).Resolve
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:310 +0x470

goroutine 34 [chan send]:
github.com/ian-kent/gopan/getpan/getpan.func��003(0xc20801ba40)
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:211 +0xaa
created by github.com/ian-kent/gopan/getpan/getpan.(*DependencyList).Resolve
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:310 +0x470

goroutine 35 [chan send]:
github.com/ian-kent/gopan/getpan/getpan.func��003(0xc20801bae0)
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:211 +0xaa
created by github.com/ian-kent/gopan/getpan/getpan.(*DependencyList).Resolve
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:310 +0x470

goroutine 36 [chan send]:
github.com/ian-kent/gopan/getpan/getpan.func��003(0xc20801a6e0)
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:211 +0xaa
created by github.com/ian-kent/gopan/getpan/getpan.(*DependencyList).Resolve
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:310 +0x470

goroutine 37 [chan send]:
github.com/ian-kent/gopan/getpan/getpan.func��003(0xc20801a820)
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:211 +0xaa
created by github.com/ian-kent/gopan/getpan/getpan.(*DependencyList).Resolve
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:310 +0x470

goroutine 38 [chan send]:
github.com/ian-kent/gopan/getpan/getpan.func��003(0xc20801af00)
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:211 +0xaa
created by github.com/ian-kent/gopan/getpan/getpan.(*DependencyList).Resolve
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:310 +0x470

goroutine 39 [chan send]:
github.com/ian-kent/gopan/getpan/getpan.func��003(0xc20801afa0)
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:211 +0xaa
created by github.com/ian-kent/gopan/getpan/getpan.(*DependencyList).Resolve
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:310 +0x470

goroutine 40 [chan send]:
github.com/ian-kent/gopan/getpan/getpan.func��003(0xc20801b0e0)
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:211 +0xaa
created by github.com/ian-kent/gopan/getpan/getpan.(*DependencyList).Resolve
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:310 +0x470

goroutine 41 [chan send]:
github.com/ian-kent/gopan/getpan/getpan.func��003(0xc20801b180)
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:211 +0xaa
created by github.com/ian-kent/gopan/getpan/getpan.(*DependencyList).Resolve
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:310 +0x470

goroutine 42 [chan send]:
github.com/ian-kent/gopan/getpan/getpan.func��003(0xc20801b1d0)
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:211 +0xaa
created by github.com/ian-kent/gopan/getpan/getpan.(*DependencyList).Resolve
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:310 +0x470

goroutine 43 [chan send]:
github.com/ian-kent/gopan/getpan/getpan.func��003(0xc20801b310)
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:211 +0xaa
created by github.com/ian-kent/gopan/getpan/getpan.(*DependencyList).Resolve
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:310 +0x470

goroutine 44 [chan send]:
github.com/ian-kent/gopan/getpan/getpan.func��003(0xc20801b450)
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:211 +0xaa
created by github.com/ian-kent/gopan/getpan/getpan.(*DependencyList).Resolve
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:310 +0x470

goroutine 45 [chan send]:
github.com/ian-kent/gopan/getpan/getpan.func��003(0xc20801b4f0)
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:211 +0xaa
created by github.com/ian-kent/gopan/getpan/getpan.(*DependencyList).Resolve
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:310 +0x470

goroutine 46 [chan send]:
github.com/ian-kent/gopan/getpan/getpan.func��003(0xc20801b630)
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:211 +0xaa
created by github.com/ian-kent/gopan/getpan/getpan.(*DependencyList).Resolve
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:310 +0x470

goroutine 47 [chan send]:
github.com/ian-kent/gopan/getpan/getpan.func��003(0xc20801b720)
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:211 +0xaa
created by github.com/ian-kent/gopan/getpan/getpan.(*DependencyList).Resolve
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:310 +0x470

goroutine 48 [chan send]:
github.com/ian-kent/gopan/getpan/getpan.func��003(0xc20801b860)
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:211 +0xaa
created by github.com/ian-kent/gopan/getpan/getpan.(*DependencyList).Resolve
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:310 +0x470

goroutine 49 [chan send]:
github.com/ian-kent/gopan/getpan/getpan.func��003(0xc20801b9f0)
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:211 +0xaa
created by github.com/ian-kent/gopan/getpan/getpan.(*DependencyList).Resolve
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:310 +0x470

goroutine 50 [chan send]:
github.com/ian-kent/gopan/getpan/getpan.func��003(0xc20801ba90)
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:211 +0xaa
created by github.com/ian-kent/gopan/getpan/getpan.(*DependencyList).Resolve
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:310 +0x470

goroutine 51 [chan send]:
github.com/ian-kent/gopan/getpan/getpan.func��003(0xc20801bb30)
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:211 +0xaa
created by github.com/ian-kent/gopan/getpan/getpan.(*DependencyList).Resolve
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:310 +0x470

goroutine 52 [chan send]:
github.com/ian-kent/gopan/getpan/getpan.func��003(0xc20801bb80)
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:211 +0xaa
created by github.com/ian-kent/gopan/getpan/getpan.(*DependencyList).Resolve
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:310 +0x470

goroutine 53 [chan send]:
github.com/ian-kent/gopan/getpan/getpan.func��003(0xc20801bbd0)
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:211 +0xaa
created by github.com/ian-kent/gopan/getpan/getpan.(*DependencyList).Resolve
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:310 +0x470

goroutine 54 [chan send]:
github.com/ian-kent/gopan/getpan/getpan.func��003(0xc20801bc70)
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:211 +0xaa
created by github.com/ian-kent/gopan/getpan/getpan.(*DependencyList).Resolve
	/Users/ikent/dev/src/github.com/ian-kent/gopan/getpan/getpan/dependency.go:310 +0x470

goroutine 59 [IO wait]:
net.runtime_pollWait(0x7f5d30c80080, 0x72, 0x0)
	/usr/local/Cellar/go/1.3.1/libexec/src/pkg/runtime/netpoll.goc:146 +0x66
net.(*pollDesc).Wait(0xc2080b6990, 0x72, 0x0, 0x0)
	/usr/local/Cellar/go/1.3.1/libexec/src/pkg/net/fd_poll_runtime.go:84 +0x46
net.(*pollDesc).WaitRead(0xc2080b6990, 0x0, 0x0)
	/usr/local/Cellar/go/1.3.1/libexec/src/pkg/net/fd_poll_runtime.go:89 +0x42
net.(*netFD).Read(0xc2080b6930, 0xc208017000, 0x1000, 0x1000, 0x0, 0x7f5d30c7e418, 0xb)
	/usr/local/Cellar/go/1.3.1/libexec/src/pkg/net/fd_unix.go:242 +0x34c
net.(*conn).Read(0xc20803e0b8, 0xc208017000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/Cellar/go/1.3.1/libexec/src/pkg/net/net.go:122 +0xe7
net/http.noteEOFReader.Read(0x7f5d30c803e0, 0xc20803e0b8, 0xc20800e318, 0xc208017000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/Cellar/go/1.3.1/libexec/src/pkg/net/http/transport.go:1203 +0x72
net/http.(*noteEOFReader).Read(0xc2080bbe20, 0xc208017000, 0x1000, 0x1000, 0xc20804c170, 0x0, 0x0)
	<autogenerated>:124 +0xca
bufio.(*Reader).fill(0xc208065f20)
	/usr/local/Cellar/go/1.3.1/libexec/src/pkg/bufio/bufio.go:97 +0x1b3
bufio.(*Reader).Peek(0xc208065f20, 0x1, 0x0, 0x0, 0x0, 0x0, 0x0)
	/usr/local/Cellar/go/1.3.1/libexec/src/pkg/bufio/bufio.go:132 +0x101
net/http.(*persistConn).readLoop(0xc20800e2c0)
	/usr/local/Cellar/go/1.3.1/libexec/src/pkg/net/http/transport.go:782 +0x95
created by net/http.(*Transport).dialConn
	/usr/local/Cellar/go/1.3.1/libexec/src/pkg/net/http/transport.go:600 +0x93f

goroutine 58 [sleep]:
time.Sleep(0x12a05f200)
	/usr/local/Cellar/go/1.3.1/libexec/src/pkg/runtime/time.goc:39 +0x31
net.func��019()
	/usr/local/Cellar/go/1.3.1/libexec/src/pkg/net/dnsclient_unix.go:183 +0x56
created by net.loadConfig
	/usr/local/Cellar/go/1.3.1/libexec/src/pkg/net/dnsclient_unix.go:212 +0x153

goroutine 60 [select]:
net/http.(*persistConn).writeLoop(0xc20800e2c0)
	/usr/local/Cellar/go/1.3.1/libexec/src/pkg/net/http/transport.go:885 +0x38f
created by net/http.(*Transport).dialConn
	/usr/local/Cellar/go/1.3.1/libexec/src/pkg/net/http/transport.go:601 +0x957
make: *** [cpan] Error 2
Build step 'Execute shell' marked build as failure
```

--- a/getpan/getpan/sources.go
+++ b/getpan/getpan/sources.go
@@ -212,10 +212,10 @@ func (s *Source) Find(d *Dependency) (*Module, error) {
 			resp, err := http.Head(archive_url)
 			if err != nil {
 				log.Trace(err)
-			} else {
-				log.Trace("HEAD status code: %d", resp.StatusCode)
+				continue
 			}
 
+			log.Trace("HEAD status code: %d", resp.StatusCode)
 			if 200 == resp.StatusCode {
 				// No module/version check since 'cpanm --info' may resolve to
 				// archive and version that may not match source
@@ -226,6 +226,7 @@ func (s *Source) Find(d *Dependency) (*Module, error) {
 					Url:     meurl,
 				}, nil
 			}
+
 		}
 		log.Error("Could not get archive URL via 'cpanm --info %s'", modver)
 		return nil, nil

